.gitignore

```diff
@@ -6,4 +6,5 @@ cdktf.out
 cdktf.log
 *terraform.*.tfstate*
 func_archive.zip
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules/
```

Comment:
This can be removed now

Reply:
[Updated](https://github.com/cdktf/cdktf-integration-serverless-python-gcp-example/pull/9/commits/6b7d5aae4ecfdc0b58925c4f1a4b3da7e5a44ce2#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R8:~:text=func_archive.zip-,func_archive.zip,.DS_Store,-node_modules/) 👍